aqueductcore/frontend/src/__mocks__/queries/getAllExtensionsMock.tsx

```diff
@@ -1,8 +1,8 @@
-import { GET_ALL_EXTENSIONS_NAME } from "API/graphql/queries/extension/getAllExtensions";
 import { ExtensionsDataMock } from "__mocks__/ExtensionsDataMock";
+import { GET_ALL_EXTENSION_NAMES } from "API/graphql/queries/getAllExtensionNames";
```

Comment:
Due to an other suggested renaming
```suggestion
import { GET_ALL_EXTENSION_NAMES } from "API/graphql/queries/extension/getAllExtensionNames";
import { ExtensionsDataMock } from "__mocks__/ExtensionsDataMock";
```